__tests__/db.test.js

```diff
@@ -34,11 +35,30 @@ describe("App", () => {
           .get("/api/topics")
           .expect(200)
           .then(({ body }) => {
-            const keys = Object.keys(body.topics[0]);
-            expect(keys.includes("slug")).toBe(true);
-            expect(keys.includes("description")).toBe(true);
+            console.log(body.topics, " <<<< my bodyyy");
+            body.topics.forEach((element) => {
```

Comment:
don't forget to make sure that any arrays you're iterating over in a test aren't empty (ie. checking the length), otherwise an empty array will give you a false positive on this test